facts/models/events.py

```diff
@@ -25,13 +26,16 @@ def save(self, *args, **kwargs):
     # def get_absolute_url(self):
     #     return reverse('facts:event-type-detail', args=[self.slug])
 
+    class Meta:
```

Comment:
These `Meta` changes don't change the database but they do require migrations for this app.